src/main/java/com/iexec/common/replicate/ReplicateStatusDetails.java

```diff
@@ -36,8 +36,7 @@ public class ReplicateStatusDetails {
     private String chainCallbackData;
     private String errorMessage;
     private ReplicateStatusCause cause;
-    private String stdout;
-    private String stderr;
+    private ReplicateLogs replicateLogs;
```

Comment:
Very good!
I would renaming field (`replicate[...]` might be useless since we are in a `ReplicateDetails`) and eventually would change type too.
`private Logs appLogs;` or `private ComputeLogs appLogs;`
?

Reply:
What about `AppComputeLogs`? This would be similar to what we already have in the SMS.

Reply:
Yes, that would be fine (better than replicate logs I think), but I'm scared someone asks for preComputeLogs and postComputeLogs. If so, we would need to re-update class name to avoid duplicates. Knowing this, what do you prefer?

Reply:
I see your point. Let's use `ComputeLogs`.

Reply:
Please see https://github.com/iExecBlockchainComputing/iexec-common/pull/326/commits/2189109bf8fc50036f9605d2d8c8d22f6df18bda :)